Mathlib/NumberTheory/EulerProduct/DirichletLSeries.lean

```diff
@@ -107,27 +105,64 @@ open scoped LSeries.notation
 
 /-- The Euler product for Dirichlet L-series, valid for `s.re > 1`.
 This version is stated in terms of `HasProd`. -/
-theorem dirichletLSeries_eulerProduct_hasProd {N : ℕ} (χ : DirichletCharacter ℂ N)
+theorem DirichletCharacter.LSeries_eulerProduct_hasProd {N : ℕ} (χ : DirichletCharacter ℂ N)
     (hs : 1 < s.re) :
     HasProd (fun p : Primes ↦ (1 - χ p * (p : ℂ) ^ (-s))⁻¹) (L ↗χ s) := by
   rw [← tsum_dirichletSummand χ hs]
   convert eulerProduct_completely_multiplicative_hasProd <| summable_dirichletSummand χ hs
 
+@[deprecated (since := "2024-11-14")] alias
+  dirichletLSeries_eulerProduct_hasProd := DirichletCharacter.LSeries_eulerProduct_hasProd
+
 /-- The Euler product for Dirichlet L-series, valid for `s.re > 1`.
 This version is stated in terms of `tprod`. -/
-theorem dirichletLSeries_eulerProduct_tprod {N : ℕ} (χ : DirichletCharacter ℂ N)
+theorem DirichletCharacter.LSeries_eulerProduct_tprod {N : ℕ} (χ : DirichletCharacter ℂ N)
     (hs : 1 < s.re) :
     ∏' p : Primes, (1 - χ p * (p : ℂ) ^ (-s))⁻¹ = L ↗χ s :=
-  (dirichletLSeries_eulerProduct_hasProd χ hs).tprod_eq
+  (DirichletCharacter.LSeries_eulerProduct_hasProd χ hs).tprod_eq
+
+@[deprecated (since := "2024-11-14")] alias
+  dirichlet_LSeries_eulerProduct_tprod := DirichletCharacter.LSeries_eulerProduct_tprod
 
 /-- The Euler product for Dirichlet L-series, valid for `s.re > 1`.
 This version is stated in the form of convergence of finite partial products. -/
-theorem dirichletLSeries_eulerProduct {N : ℕ} (χ : DirichletCharacter ℂ N) (hs : 1 < s.re) :
+theorem DirichletCharacter.LSeries_eulerProduct {N : ℕ} (χ : DirichletCharacter ℂ N)
+    (hs : 1 < s.re) :
     Tendsto (fun n : ℕ ↦ ∏ p ∈ primesBelow n, (1 - χ p * (p : ℂ) ^ (-s))⁻¹) atTop
       (𝓝 (L ↗χ s)) := by
   rw [← tsum_dirichletSummand χ hs]
   apply eulerProduct_completely_multiplicative <| summable_dirichletSummand χ hs
 
+@[deprecated (since := "2024-11-14")] alias
+  dirichletLSeries_eulerProduct := DirichletCharacter.LSeries_eulerProduct
+
+open LSeries
+
+/-- A variant of the Euler product for Dirichlet L-series. -/
+theorem DirichletCharacter.LSeries_eulerProduct' {N : ℕ} (χ : DirichletCharacter ℂ N) {s : ℂ}
```

Comment:
What do you think of adding `_explog` as a suffix, instead of `'`.
```suggestion
theorem DirichletCharacter.LSeries_eulerProduct_explog {N : ℕ} (χ : DirichletCharacter ℂ N) {s : ℂ}
```
It is a bit longer, but it also conveys what the variant is...

Idem dito for the decls below.

Reply:
Sounds good; will do.

Reply:
(using `..._exp_log`; this looks closer to the naming convention.)